app/screens/home/recent_mentions/index.tsx

```diff
@@ -29,20 +102,105 @@ const RecentMentionsScreen = () => {
             opacity: withTiming(0, {duration: 150}),
             transform: [{translateX: withTiming(toLeft ? -25 : 25, {duration: 150})}],
         };
-    }, [start]);
+    }, [isFocused]);
```

Comment:
actually this is causing the `useAnimatedStyle` to create a new worklet every time the prop changes it's value, there is a better way to approach this by having the deps as an empty array and use a shared value instead that would be 0 or 1 depending on `isFocused` something like this.

```typescript
const opacity = useSharedValue(isFocused ? 1 : 0);
const translateSide = toLeft ? - 25 : 25;
const translateX = useSharedValue(isFocused ? 0 : translateSide);

...

const animated = useAnimatedStyle(() => {
    return {
        opacity: withTiming(opacity.value, {duration: 150}),
        transform: [{translateX: withTiming(translateX.value, {duration: 150})}],
    };
}, []);

useEffect(() => {
    opacity.value = isFocused ? 0 : 1;
    translateX.value = isFocused ? 0 : translateSide;
}, [isFocused]);
```